pkg/cloud/common/config_test.go

```diff
@@ -0,0 +1,61 @@
+package common
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	infraCloudConfName = "test-config"
+	infraCloudConfKey  = "foo"
+)
+
+func makeInfrastructureResource(platform configv1.PlatformType) *configv1.Infrastructure {
+	return &configv1.Infrastructure{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster",
+		},
+		Spec: configv1.InfrastructureSpec{
+			CloudConfig: configv1.ConfigMapFileReference{
+				Name: infraCloudConfName,
+				Key:  infraCloudConfKey,
+			},
+			PlatformSpec: configv1.PlatformSpec{
+				Type: platform,
+			},
+		},
+	}
+}
+
+func makeInfraCloudConfig() *corev1.ConfigMap {
+	return &corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{
+		Name:      infraCloudConfName,
+		Namespace: "openshift-config",
+	}, Data: map[string]string{infraCloudConfKey: "bar"}}
+}
+
+var _ = Describe("NoOpTransformer method", func() {
+	infra := makeInfrastructureResource(configv1.AzurePlatformType)
+	infraCloudConfig := makeInfraCloudConfig()
```

Comment:
In case any of your tests modify these, it's better to ensure they are freshly initialised on a per test basis

```suggestion
	var infra *configv1.Infrastructure
	var infraCloudConfig *corev1.ConfigMap

	BeforeEach(func() {
		infra = makeInfrastructureResource(configv1.AzurePlatformType)
		infraCloudConfig = makeInfraCloudConfig()
	})
```